src/components/NetworkSettings/NetworkStatusBar.js

```diff
@@ -5,26 +5,28 @@
  * LICENSE file in the root directory of this source tree.
  */
 import { useSelector } from 'react-redux';
-import { eq } from 'lodash';
+import { isEqual } from 'lodash';
 import { t } from 'ttag';
 import { AlertUI } from '../../styles/themes';
 import { ToplineBar } from '../ToplineBar';
 import { PRE_SETTINGS_MAINNET } from '../../constants';
 
 const customNetworkText = t`Custom network`;
 
+function notMainnet(networkSettings) {
+  return !isEqual(networkSettings, PRE_SETTINGS_MAINNET);
```

Comment:
I think the mainnet warning bug is here, we should exclude the wallet service URLs when comparing the settings.

Reply:
What bug?

Reply:
Ok. After our talk I understood. I changed it to ignore the fields of wallet service URL: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/360/commits/51609ad2c25538dff76d21b84addb3151fca275c